.github/workflows/molecule.yml

```diff
@@ -0,0 +1,29 @@
+name: Test with Molecule
+on:
+  workflow_call:
+    inputs:
+      run-tags:
+        required: false
+        type: string
+        default: all
```

Comment:
`required` isn't required if a `default` is set

Reply:
well spotted, I've removed it